src/GNDStk/child_t/src/operator.hpp

```diff
@@ -100,11 +90,29 @@ inline auto operator--(const child_t<TYPE,FIND,CONVERTER> &kwd, const int)
    );
 }
 
-// child_t<TYPE>++
+// operator++
```

Comment:
I'm not sure what these operators do.

Reply:
These, as well as the division operators, comprise (part of, for now) what I believe might prove to be a handy, albeit somewhat stylized and probably unexpected, "keyword arithmetic" system I have in mind. I thought you might ask about it.

Without going into too much detail here... Recall the examples I've shown (when I first described it, and also during my WPEC talk) about using what I've called my keyword system: the `child_t` and `meta_t` types that embed information, in particular something's name in GNDS, and a *type* with which you want it to interact.

Basically, for `child_t` and `meta_t`, `operator/` as well as `++` and `--` are designed so that you can take an already-made `child_t` or `meta_t` and easily create another similar one - based mostly on the original one, but with a small modification. Just as one example: let's say you have a `child_t` called `mydate` that's encoded to give you some custom type `date_t`. Then you decide you also want to allow for another format, `date2_t`. Write `auto mydate2 = date2_t{}/mydate`, and it does just that: duplicate everything from `mydate`, except change `<date_t>` somewhere to `<date2_t>`.

Remember the `one` versus `all` business? The `--` and `++` operators, applied to a `child_t`, "upgrade" to `all` or "downgrade" to `one`. (In doing so they actually create a new `child_t`, they don't modify the existing one as `int++` modifies the int in question. The language does allow this.)

In short, it's a rather stylized concept, but one I think will be helpful. It'll especially be helpful if and when I expand the capabilities of `child_t` and `meta_t` - which I plan to do. (@jlconlin in fact you yourself suggested when we first spoke that `child_t` could be outfitted with a "filter" or sorts, so that it would only pull out child nodes, of the given name, that satisfied some user-defined condition. So, imagine going to the syntactic effort to set up such an animal. Then, my keyword-arithmetic system will allow you to easily made other `child_t` objects from the first one, with less effort, merely with some expression like what I wrote a couple of paragraphs above.)